src/tests/suite/daos_rebuild_ec.c

```diff
@@ -40,6 +40,7 @@ rebuild_ec_internal(void **state, daos_oclass_id_t oclass, int kill_data_nr,
 	if (svc_nreplicas < 5)
 		return;
 
+	daos_pool_set_prop(arg->pool.pool_uuid, "reclaim", "disabled");
```

Comment:
why do we disable aggregation?   Is it intended to isolate bugs?  I worry it might be hiding some if that's the case?

Reply:
oh, the test case suppose to verify the case with partial update + full stripe update, 

"enable aggregation" will make partial update + full stripe update being aggregated with EC + VOS aggregation,  which can make the test false pass.  

And this only disable aggregation for one third of the EC rebuild tests,  and the other two thirds still run with aggregation enable.